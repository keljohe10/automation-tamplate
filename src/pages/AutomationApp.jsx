import React from 'react';
import { BrowserRouter as Router } from 'react-router-dom';

import Container from 'emerald-ui/lib/Container';
import Row from 'emerald-ui/lib/Row';
import Col from 'emerald-ui/lib/Col';

import Navbar from "../components/common/navbar/Navbar";
import AppRouter from "../routers/AppRouter";

import './styles/styles.css';

const AutomationApp = () => {
    return (
        <Router>
            <Container fluid={true}>
                <Row>
                    <Col xs={2} className="sample-col app-row-col app-col-navbar">
                        <Navbar />
                    </Col>
                    <Col xs={10} className="sample-col app-row-col">
                        <AppRouter />
                    </Col>
                </Row>
            </Container>
        </Router>
    )
}

export default AutomationApp;
