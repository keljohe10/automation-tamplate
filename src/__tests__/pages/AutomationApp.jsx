import React from 'react';
import '@testing-library/jest-dom';
import { shallow } from 'enzyme';
import AutomationApp from '../../pages/AutomationApp';

describe('When load the automation pages', () => {
    const wrapper = shallow(<AutomationApp />);

    test('Should render the AutomationApp', () => {

        expect(wrapper).toMatchSnapshot();
    });


});
