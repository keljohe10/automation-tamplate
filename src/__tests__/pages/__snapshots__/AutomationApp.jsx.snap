// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`When load the automation pages Should render the AutomationApp 1`] = `
<BrowserRouter>
  <Container
    displayName="Container"
    fluid={true}
  >
    <Row>
      <Col
        className="sample-col app-row-col app-col-navbar"
        xs={2}
      >
        <Navbar />
      </Col>
      <Col
        className="sample-col app-row-col"
        xs={10}
      >
        <AppRouter />
      </Col>
    </Row>
  </Container>
</BrowserRouter>
`;
